### DISCUSSION TITLE
FAST_Solution:FAST_AdvanceStates:AD_UpdateStates:SetInputs:TwrInfl:getLocalTowerProps:Tower strike.

### DISCUSSION THREAD

**Discussion by LeoHongLiZuo** [OpenFAST/openfast#1974] [Q&A]
Dear guys,
When I run the IEA15MW wind turbine.
If I set it to 10m/s steady wind, Openfast is running properly.
But when I switched to a 10m/s wind file generated by Turbsim, an error will be reported like this.
![微信图片_20240112092625](https://github.com/OpenFAST/openfast/assets/108517989/5e96fd42-f4b9-43ea-ab5a-a57678f0e8d2)
I don't know why the Tower strike after changing to wind files generated by Turbsim.
And I am not sure how to solve this problem according to this solution.
Could someone please tell me how to solve it?
Thanks a lot!
Leo

**Comment by Steven647** [comment]
Is the original 15MW turbine model being used, and have there been any changes to the parameters of tower?

**Reply by LeoHongLiZuo** [reply]
Yes, and I haven't changed the original parameters

**Reply by LeoHongLiZuo** [reply]
If it's convenient for you, I think we can add a wechat,13314719037

**Reply by jjonkman** [reply]
Dear @LeoHongLiZuo,

You are receiving several other warnings before the tower strike error that suggest other issues with your simulation.  First, have you set proper initial conditions of the rotor speed and blade-pitch angles conditioned on the 10-m/s mean hub-height wind speed you are simulating?

Best regards,

**Reply by LeoHongLiZuo** [reply]
Dear @jjonkman,
I have set initial rotor speed and blade-pitch angles in ElastoDyn . But it's still Tower strike.
I don't know why does steady wind work with the same 10m/s wind, but Turbsim generated turbulent wind will report errors.
Best regards,

**Reply by jjonkman** [reply]
Dear @LeoHongLiZuo,

Can you share plots of the response up until the tower strike error is triggered?  What does the wind speed, rotor speed, blade pitch, blade deflections, and tower deflections look like?

Best regards,

**Reply by LeoHongLiZuo** [reply]
Dear @jjonkman
The error was resolved after I tried a different the blade-pitch angles and set rotor speed according to the instructions file.
Thank you very much for your advice and help!
Best regards,

**Comment by LeoHongLiZuo** [comment]
resolved
